packages/synthchron-simulator/src/types/processModelTypes/petriNetTypes.ts

```diff
@@ -10,13 +10,17 @@ export interface PetriNetPlace {
   name: string
   accepting: (numOfTokens: number) => boolean
   amountOfTokens: number
+  x?: number
+  y?: number
 }
 
 export interface PetriNetTransition {
   type: 'transition'
   identifier: string
   weight: number
   name: string
+  x?: number
+  y?: number
```

Comment:
I think we should extract the position into another type. Otherwise, we'll have to duplicate the code for all possible nodes.

Reply:
Yeah wouldnt it be able to have some default, since every node should have x and y, and the derive it in every node interface? Or are you thinking of a Position type for each node, but comes with the same problems, so i assume the first is what u are talking about?